compiler/optimizer/VPConstraint.hpp

```diff
@@ -961,6 +961,7 @@ class VPKnownObject : public TR::VPFixedClass
 
    virtual TR_YesNoMaybe isJavaLangClassObject();
    virtual bool isArrayWithConstantElements(TR::Compilation * comp);
+   virtual bool isArrayWithStableElements(TR::Compilation * comp);
```

Comment:
This doesn't need to be virtual, but neither does the existing `isArrayWithConstantElements()`

Reply:
fixed